### PR TITLE
Add links to manuals

### PR DESCRIPTION
**Pull request by brianamarie**
## Overview
**TL;DR**

We are open sourcing the training manuals. This PR adds links to those manuals, so that the `training-kit` landing page is a resource for finding other learning content in addition to cheat sheets.

### Questions
<If relevant, write a list of questions that you would like to discuss related to the changes that you have made.>
N/A

### Next Steps
<If incomplete, create a task list of items that are still being worked on within the Pull Request.>

As we move the training manual for developers into the GitHub org, we will need to update the link here. 

### Review
<If complete, or ready for :eyes:, use @mentions for quick questions, specific feedback, and progress updates.>

cc @parkerbxyz @github/ps-programs for awareness and review 👀

### CODE REVIEW

**Comment by hollenberry** [comment]
🎉 🎉 🎉

**Comment by brianamarie** [comment]
@github/ps-programs @parkerbxyz Would anyone have a few minutes to review this? Once this is merged, I think we can build on it with #765, and also configure a redirect from training.github.com to the landing page for training kit.

**Review by parkerbxyz — APPROVED** [review]

**Comment by Jay-Tarzan** [comment]
click click - double feature click - wrong, false
